tests/net/buf/src/main.c

```diff
@@ -731,6 +731,38 @@ ZTEST(net_buf_tests, test_net_buf_user_data)
 	net_buf_unref(buf);
 }
 
+ZTEST(net_buf_tests, test_net_buf_user_data_clone)
+{
+	struct net_buf *buf_a, *buf_b;
+	uint32_t *src_user_data, *dst_user_data;
+
+	buf_a = net_buf_alloc_len(&bufs_pool, 1, K_NO_WAIT);
+	zassert_not_null(buf_a, "Failed to get buffer");
+	src_user_data = net_buf_user_data(buf_a);
+	*src_user_data = 0xAABBCCDD;
+
+	/* Happy case: User data size of destination buffer exceeds the on in the source buffer */
```

Comment:
I probably missed something here but what does this mean
`...exceeds the on in the source buffer`
or is there a typo in the comment?

Reply:
Should have be "one" instead of "on". Will fix.

Reply:
Improved. Not sure if 100% understandable. I'm not a native speaker, open for better suggestions.